frontend/src/metabase/components/ErrorPages/utils.ts

```diff
@@ -1,3 +1,5 @@
+import Bowser from "bowser";
```

Comment:
I'm curious if we really need this dependency. Fwiw we have some functions already that inspect the user agent, like `isMac()` and `navigatorInfo()` (which is in Help.tsx but currently not exported).

Reply:
Are you thinking that Bowser is the best way to extract the OS version?

Reply:
Not a blocker. It sparked a useful conversation about bundle size, though: https://metaboat.slack.com/archives/C0669P4AF9N/p1733841293997639.